tensorrt_llm/_torch/pyexecutor/_util.py

```
@@ -220,9 +220,15 @@ def _get_token_num_for_estimation(self) -> int:
         # If not able to allocate self._model_engine.batch_size blocks, the max batch size should be adjusted.
         num_cache_blocks = max(num_cache_blocks, self._model_engine.batch_size)
 
+        mem_free, mem_total = torch.cuda.mem_get_info()
```

Ruff (0.14.1)

Unpacked variable `mem_total` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)